pkg/action/actioncontroller.go

```diff
@@ -121,7 +127,13 @@ func (ac *ActionController) handle(ctx context.Context, action *storkv1.Action)
 		case storkv1.ActionStageInitial:
 			ac.verifyMigrationScheduleBeforeFailback(action)
 		case storkv1.ActionStageScaleDownDestination:
-			ac.performDeactivateFailBackStage(action)
+			ac.performDeactivateFailBackStageDestination(action)
```

Comment:
Would `performDeactivateDestinationFailBackStage` or `performFailBackDeactivateDestinationStage` be better names than `performDeactivateFailBackStageDestination`? 
Similarly for rest of the functions.